CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+## I found a bug!
+
+### 1. Search for it
+[Briefly search the issues](https://github.com/KittehOrg/KittehIRCClientLib/issues)
+to see if there is an existing issue covering this bug.
+Respond as you feel appropriate, if there is one.
+
+### 2. Create a new issue
+
+[Create an issue](https://github.com/KittehOrg/KittehIRCClientLib/issues/new)!
+Be sure to include as much of the following as you can:
+
+ * Version of the client library.
```

Comment:
Can be psuedo enforced by adding a [`.github/ISSUE_TEMPLATE/Bug_Report.md`](https://github.com/jkcclemens/paste/tree/master/.github/ISSUE_TEMPLATE)

Reply:
Thanks! I'll do that.